kdl-client/src/Pages/Server/pages/Server/components/ServerBar/components/Breadcrumbs/Breadcrumbs.module.scss

```
@@ -0,0 +1,16 @@
+@import 'Styles/variables';
+@import 'Styles/mixins';
+@import 'Styles/colors';
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  & > div {
```

I think & is not necessary here

```suggestion
  > div {
```